in-page-scripts/integrations/GitLab.ts

```diff
@@ -0,0 +1,119 @@
+﻿/// <reference path="../../interfaces/integrations" />
+/// <reference path="../IntegrationService" />
+/// <reference path="../utils" />
+
+module Integrations {
+    class GitLab implements WebToolIntegration {
+
+        matchUrl = [
+            '*://*/issues/*',
+            '*://*/merge_requests/*'
+        ];
+
+        match(source: Source): boolean {
+            var descriptionMeta = $$('meta[name=description]');
+            if (descriptionMeta) {
+                return descriptionMeta.getAttribute('content').indexOf('GitLab') > -1;
+            }
+            return false;
+        }
+
+        render(issueElement: HTMLElement, linkElement: HTMLElement) {
+            var issueHost = $$('.issue .issue-details .page-title .pull-right');
```

Comment:
Selectors are too complex. Use '.page-title .pull-right' in both cases. Moreover, as fallback append link to the end of '.page-title'.